app/src/main/res/layout/activity_session_feedback.xml

```diff
@@ -23,234 +23,295 @@
-                android:text="@string/session_feedback_relevance"
-                android:textAlignment="viewStart">
-
-                <requestFocus />
-            </TextView>
-
-            <io.github.droidkaigi.confsched.widget.NumberRatingBar
-                android:id="@+id/relevant_feedback_bar"
-                style="@style/FeedbackNumberRatingBar"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/spacing_small"
-                android:layout_marginLeft="@dimen/spacing_small"
-                android:layout_marginRight="@dimen/spacing_small"
-                android:layout_marginStart="@dimen/spacing_small"
-                android:maxHeight="?android:listPreferredItemHeightSmall"
-                android:minHeight="?android:listPreferredItemHeightSmall"
-                android:paddingTop="@dimen/spacing_small" />
-
-            <FrameLayout
+            <android.support.v7.widget.CardView
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:paddingBottom="@dimen/spacing"
-                android:paddingEnd="@dimen/spacing"
-                android:paddingLeft="@dimen/spacing"
-                android:paddingRight="@dimen/spacing"
-                android:paddingStart="@dimen/spacing"
-                android:paddingTop="@dimen/spacing_xsmall">
+                style="@style/FeedbackCardItem">
 
-                <TextView
-                    style="@style/FeedbackQuestionGuideText"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="start|left"
-                    android:text="@string/session_feedback_not_at_all_relevant" />
-
-                <TextView
-                    style="@style/FeedbackQuestionGuideText"
-                    android:layout_width="wrap_content"
+                <LinearLayout
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_gravity="end|right"
-                    android:text="@string/session_feedback_extremely_relevant" />
-
-            </FrameLayout>
+                    android:orientation="vertical">
+
+                    <TextView
+                        style="@style/FeedbackTitle"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:focusable="true"
+                        android:focusableInTouchMode="true"
+                        android:gravity="center_vertical"
+                        android:minHeight="?android:listPreferredItemHeightSmall"
+                        android:text="@string/session_feedback_relevance"
+                        android:textAlignment="viewStart">
```

Comment:
It is original issue. `textAlignment` is conflict with `gravity` attribute. So it is better to remove `textAlignment`, and add gravity `start`.

```xml
android:gravity="center_vertical | start"
```